Libraries_API.ipynb

```diff
@@ -713,6 +664,151 @@
+    "The following type of user can add or remove documents:\n",
+    "  * owner\n",
+    "  * admin\n",
+    "  * write\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current\n",
+      "                                 Dload  Upload   Total   Spent    Left  Speed\n",
+      "100   342  100   265  100    77     59     17  0:00:04  0:00:04 --:--:--    77\n"
+     ]
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "{\"number_added\": 10, \"bibcodes\": [\"1973PhRvD...7.2333B\", \"2023MNRAS.518.1057E\", \"1977MNRAS.179..433B\", \"2016PhRvL.116f1102A\", \"1975CMaPh..43..199H\", \"1976PhRvD..14..870U\", \"2013ARA&A..51..511K\", \"2022RAA....22l2002S\", \"1973A&A....24..337S\", \"1974Natur.248...30H\"]}\n"
+     ]
+    }
+   ],
+   "source": [
+    "%%bash\n",
+    "token=\"your-token-here\"\n",
+    "# add a query by posting search parameters\n",
+    "curl -H \"Authorization: Bearer $token\" -H \"Content-Type: application/json\" \\\n",
+    "    https://api.adsabs.harvard.edu/v1/biblib/query/6TI2c01jSYe1WT0xNmw-vw \\\n",
+    "    -X POST \\\n",
+    "    -d '{\"params\": {\"q\": \"black holes\", \"fq\": \"database:astronomy\"}, \"action\": \"add\"}'"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 13,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current\n",
+      "                                 Dload  Upload   Total   Spent    Left  Speed\n",
+      "100   347  100   267  100    80    978    293 --:--:-- --:--:-- --:--:--  1299\n"
+     ]
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "{\"number_removed\": 10, \"bibcodes\": [\"1973PhRvD...7.2333B\", \"2023MNRAS.518.1057E\", \"1977MNRAS.179..433B\", \"2016PhRvL.116f1102A\", \"1975CMaPh..43..199H\", \"1976PhRvD..14..870U\", \"2013ARA&A..51..511K\", \"2022RAA....22l2002S\", \"1973A&A....24..337S\", \"1974Natur.248...30H\"]}\n"
+     ]
+    }
+   ],
+   "source": [
+    "%%bash\n",
+    "token=\"your-token-here\"\n",
+    "# remove the bibcode by query\n",
+    "curl -H \"Authorization: Bearer $token\" -H \"Content-Type: application/json\" \\\n",
+    "    https://api.adsabs.harvard.edu/v1/biblib/query/6TI2c01jSYe1WT0xNmw-vw \\\n",
+    "    -X POST \\\n",
+    "    -d '{\"params\": {\"q\": \"black holes\", \"fq\": \"database:astronomy\"}, \"action\": \"remove\"}'"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "\n",
+    "#### GET\n",
+    "Add a document to a given library. Returns the number of documents added along with a list of added bibcodes\n",
```

Comment:
I'd say explicitly that this endpoint is only for adding bibcodes to a library, and that to remove bibcodes via query you need to use the POST endpoint